chart/templates/worker/first-rows/_container.tpl

```diff
@@ -34,6 +34,8 @@
     value: {{ .Values.firstRows.minCellBytes | quote }}
   - name: FIRST_ROWS_MIN_NUMBER
     value: {{ .Values.firstRows.minNumber| quote }}
+  - name: FIRST_ROWS_COLUMNS_MAX_NUMBER
+    value: {{ .Values.firstRows.columns_max_number| quote }}
```

Comment:
columns_max_number -> columnsMaxNumber

And you also have to add the value in values.yaml, in the worker template, and if necessary in prod.yaml and dev.yaml

Reply:
Thanks, I will add the it to the templates